### PR TITLE
Feature/si 464 system status manager role

### PR DESCRIPTION
**Pull request by hutnikau**
Restrict access to the extension.
Only taoSystemStatusManager role should has access to taoSystemStatus extensions

### CODE REVIEW

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Updater extends common_ext_ExtensionUpdater
      */
     public function update($initialVersion)
     {
-        $this->skip('0.0.1', '0.9.0');
+        $this->skip('0.0.1', '0.10.0');
```
Comment (zagovorichev) [inline scripts/update/Updater.php]: You have to update the role in the updater as well

**Review by zagovorichev — CHANGES_REQUESTED** [review]

File: scripts/update/Updater.php (hunk 1 above)
Comment (hutnikau) [reply scripts/update/Updater.php]: This extension is not used on prod yet. So it's not necessary

File: model/Ontology/taosystemstatus.rdf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<rdf:RDF 
+	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
+	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" 
+	xml:base="http://www.tao.lu/Ontologies/TAOTestCenter.rdf#"
```
Comment (zagovorichev) [inline model/Ontology/taosystemstatus.rdf]: not clear how have you managed this RDF,
I guess you need to create `xml:base="http://www.tao.lu/Ontologies/TAOSystemStatus.rdf#"`
and then
`http://www.tao.lu/Ontologies/TAOSystemStatus.rdf#taoSystemStatusManager`

**Review by zagovorichev — CHANGES_REQUESTED** [review]

File: scripts/update/Updater.php (hunk 1 above)
Comment (zagovorichev) [reply scripts/update/Updater.php]: yep, I see, but this is possible that developers will have an issue with roles in the future

Comment (hutnikau) [reply scripts/update/Updater.php]: Done. Please review it again.

**Review by zagovorichev — APPROVED** [review]

**Pull request merged**